### DISCUSSION TITLE
Start implementing rspec/rubocop

### DISCUSSION THREAD

**Discussion by madoleary** [tosdr/edit.tosdr.org#1044] [Ideas]
Think it'd be a good idea to start slowly implementing some tests. Rubocop is good to have too, to stay on top of Rails conventions and Ruby syntax best practices.

**Comment by evilcel3ri** [comment]
On test I deeply, strongly agree. And on rubucop as well. So we can install some continuous integration and it will save us a lot of time in the development and the deployment.

For the test, I propose something like:
### Test user
* Test user logs in
* Test user creates a point  
* Test user modifies its point 
* Test user destroy its point
* Test user is destroyed

### Curator Test user
* Curator Test user creates a service
* Curator Test user creates a point on the service
* Curator Test user modify this points on this services
* Curator Test user looking into pending points and modify the status of a point
* Curator Test is destroyed

What do you think ?

**Comment by madoleary** [comment]
Agree on all fronts

**Comment by evilcel3ri** [comment]
Notes for later. We might want to look into [Travis ](http://docs.travis-ci.com/)that is free for open source and directly integrated into Github. We can try it first for styling.
